src/components/headers_overlay/headers_overlay.ts

```diff
@@ -200,6 +200,10 @@ abstract class AbstractResizer extends Component<ResizerProps, SpreadsheetChildE
       this.startMovement(ev);
       return;
     }
+    if (this.env.model.getters.getEditionMode() === "editing") {
+      this.env.model.dispatch("UNFOCUS_SELECTION_INPUT");
+      this.env.model.dispatch("STOP_EDITION");
+    }
```

Comment:
as discussed, we should add a possibility on the selection processor to kick the active handler, whatever it is.
This part should not know any detail about who it is and how it works